localstack/services/dynamodbstreams/dynamodbstreams_api.py

```diff
@@ -117,5 +126,18 @@ def stream_name_from_stream_arn(stream_arn):
     return get_kinesis_stream_name(table_name)
 
 
+def random_id(stream_arn, kinesis_shard_id, n=16):
```

Comment:
Looks like we're not using the `n=16` parameter. Also, is there a specific reason why we're using an explicit namespace instead of simply using `uuid4` for example? We could simplify this function a bit:
```
def random_id():
    return str(uuid.uuid4())
```
(or even remove this function and use the value directly in line 135).

Reply:
Good catch on `n`.  I will remove it. 

I wanted successive calls to `describe_stream` to follow the Kinesis topology.  In other words, if the topology of the underlying stream didn't change I wanted to same ShardIDs to be returned by `describe_stream`.   This is why I opted to use uuid5 relative to StreamID and the ShardID.

Reply:
`n` has been removed.

Reply:
Right, good point, I missed that part. Thanks for clarifying. On second thought - since the UUID is derived from StreamID and ShardID, we could have simply appended a fixed/constant string. But I think what we have now is fine.